pkg/kubectl/stop.go

```diff
@@ -509,3 +507,7 @@ func (reaper *ServiceReaper) Stop(namespace, name string, timeout time.Duration,
 	}
 	return services.Delete(name)
 }
+
+func getNonOrphanDeleteOptions() *api.DeleteOptions {
+	return &api.DeleteOptions{OrphanDependents: new(bool)}
```

Comment:
@krmayankk thanks for the comment. From https://golang.org/doc/effective_go.html#allocation_new: `new(T)` returns a `*T` pointer to a zero value of type `T`. In this case, `new(bool)` returns a `*bool` pointer to `false`. I don't think it's not readable, and the function name also tells you what'll be returned.